fiscalsim_us/parameters/gov/states/mt/tax/income/deductions/standard/min.yaml

```diff
@@ -14,17 +19,22 @@ metadata:
       href: https://mtrevenue.gov/wp-content/uploads/mdocs/form%202%202021.pdf#page=7
 
 JOINT:
+  #increase each year with inflation (repealed effective January 1, 2024)
```

Comment:
@finpaul. I would like to have this statement in this file just one time. I don't think it belongs in the references, but maybe it does. Let's think about where to put it--either the top of or the bottom.